src/applications/mhv-secure-messaging/tests/e2e/utils/constants.js

```diff
@@ -408,6 +409,14 @@ export const Data = {
     SIX_MONTHS: `Last 6 months`,
     TWELVE_MONTHS: `Last 12 months`,
   },
+  INBOX_FOOTER: {
+    HEADER: `Need help?`,
+    PARAGRAPH_1: `Have questions about how messages works?`,
+    PARAGRAPH_2: `Want to send a message to a care team that’s not on your list? Contact your VA health facility. Ask for the My HealtheVet coordinator or secure messaging administrator.`,
+
+    LINK_1: `Learn more about messages`,
+    LINK_2: `Find your VA health facility`,
+  },
```

Comment:
@fazilqa I meant to ask this awhile ago. But since we have an opportunity here - what is the benefit of maintaining separate set of constants  if they have the same values as in `src/applications/mhv-secure-messaging/util/constants.js` ? This is not necessarily a directive to change this, at least not now, if there is a good reason that I am probably not thinking of. I would like to be educated on this

Reply:
@fazilqa thank you. Let's consider having a task for the future for consolidating that